src/Partial.js

```diff
@@ -2,7 +2,7 @@
 
 // module Partial
 
-exports.crashWith = function () {
+exports._crashWith = function () {
```

Comment:
The outer function can be removed here now that the constraint is gone from the foreign type.